src/main/java/decodex/recipes/RecipeManager.java

```diff
@@ -0,0 +1,73 @@
+package decodex.recipes;
+
+import decodex.Decodex;
+import decodex.data.exception.RecipeManagerException;
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+/**
+ * RecipeManager manages a list of all recipes that were created/loaded.
+ */
+public class RecipeManager {
+
+    private Logger logger = Decodex.logger;
+
+    ArrayList<Recipe> recipeList;
+
+    /**
+     * Creates a new RecipeManager with no recipes.
+     */
+    public RecipeManager() {
+        this.recipeList = new ArrayList<>();
+    }
+
+    /**
+     * Creates a new RecipeManager with a predefined list of recipes.
+     *
+     * @param recipeList list of recipes to be loaded.
+     */
+    public RecipeManager(ArrayList<Recipe> recipeList) {
```

Comment:
Consider the scenario when no recipes exist, like when the program is first run. Passing in an empty `recipeList` is redundant.

Reply:
Agreed with @arraysius, probably can just enforce that the program has to append to an existing `RecipeManager`.